NPE when stopping JDT.LS

with `redhat.java@v1.12.2022100704`, when reloading VS Code, errors can be observed in server log

```
!ENTRY org.eclipse.jdt.ls.core 1 0 2022-10-08 13:01:14.005
!MESSAGE class org.eclipse.jdt.ls.core.internal.JavaLanguageServerPlugin is stopping:

!ENTRY com.microsoft.java.debug.plugin 4 0 2022-10-08 13:01:15.002
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Exception in com.microsoft.java.debug.plugin.internal.JavaDebuggerServerPlugin.stop() of bundle com.microsoft.java.debug.plugin.
	at org.eclipse.osgi.internal.framework.BundleContextImpl.stop(BundleContextImpl.java:895)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.stopWorker0(EquinoxBundle.java:1046)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.stopWorker(EquinoxBundle.java:376)
	at org.eclipse.osgi.container.Module.doStop(Module.java:660)
	at org.eclipse.osgi.container.Module.stop(Module.java:521)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.decStartLevel(ModuleContainer.java:1893)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1768)
	at org.eclipse.osgi.container.SystemModule.stopWorker(SystemModule.java:275)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$SystemBundle$EquinoxSystemModule.stopWorker(EquinoxBundle.java:208)
	at org.eclipse.osgi.container.Module.doStop(Module.java:660)
	at org.eclipse.osgi.container.Module.stop(Module.java:521)
	at org.eclipse.osgi.container.SystemModule.stop(SystemModule.java:207)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$SystemBundle$EquinoxSystemModule$1.run(EquinoxBundle.java:226)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NullPointerException: Cannot invoke "org.eclipse.equinox.log.Logger.log(Object, int, String, java.lang.Throwable)" because "this.logger" is null
	at org.eclipse.core.internal.runtime.Log.log(Log.java:68)
	at com.microsoft.java.debug.plugin.internal.JdtLogHandler.publish(JdtLogHandler.java:48)
	at java.logging/java.util.logging.Logger.log(Unknown Source)
	at java.logging/java.util.logging.Logger.doLog(Unknown Source)
	at java.logging/java.util.logging.Logger.log(Unknown Source)
	at java.logging/java.util.logging.Logger.info(Unknown Source)
	at com.microsoft.java.debug.plugin.internal.JavaDebuggerServerPlugin.stop(JavaDebuggerServerPlugin.java:37)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:875)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:1)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.stop(BundleContextImpl.java:867)
	... 13 more
Root exception:
java.lang.NullPointerException: Cannot invoke "org.eclipse.equinox.log.Logger.log(Object, int, String, java.lang.Throwable)" because "this.logger" is null
	at org.eclipse.core.internal.runtime.Log.log(Log.java:68)
	at com.microsoft.java.debug.plugin.internal.JdtLogHandler.publish(JdtLogHandler.java:48)
	at java.logging/java.util.logging.Logger.log(Unknown Source)
	at java.logging/java.util.logging.Logger.doLog(Unknown Source)
	at java.logging/java.util.logging.Logger.log(Unknown Source)
	at java.logging/java.util.logging.Logger.info(Unknown Source)
	at com.microsoft.java.debug.plugin.internal.JavaDebuggerServerPlugin.stop(JavaDebuggerServerPlugin.java:37)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:875)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:1)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.stop(BundleContextImpl.java:867)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.stopWorker0(EquinoxBundle.java:1046)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.stopWorker(EquinoxBundle.java:376)
	at org.eclipse.osgi.container.Module.doStop(Module.java:660)
	at org.eclipse.osgi.container.Module.stop(Module.java:521)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.decStartLevel(ModuleContainer.java:1893)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1768)
	at org.eclipse.osgi.container.SystemModule.stopWorker(SystemModule.java:275)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$SystemBundle$EquinoxSystemModule.stopWorker(EquinoxBundle.java:208)
	at org.eclipse.osgi.container.Module.doStop(Module.java:660)
	at org.eclipse.osgi.container.Module.stop(Module.java:521)
	at org.eclipse.osgi.container.SystemModule.stop(SystemModule.java:207)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$SystemBundle$EquinoxSystemModule$1.run(EquinoxBundle.java:226)
	at java.base/java.lang.Thread.run(Unknown Source)
```

--- a/com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/JavaDebuggerServerPlugin.java
+++ b/com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/JavaDebuggerServerPlugin.java
@@ -34,7 +34,6 @@ public class JavaDebuggerServerPlugin implements BundleActivator {
 
     @Override
     public void stop(BundleContext context) throws Exception {
-        logger.info("Stopping " + PLUGIN_ID);
         LogUtils.cleanupHandlers();
     }
 

